eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -50,6 +64,28 @@ foreach ($branch in $branches)
     continue
   }
 
+  if ($LastCommitOlderThan -or $env:LASTCOMMITDATEOVERRIDE) {
+    $LastCommitOlderThan = If (!$LastCommitOlderThan) { $env:LASTCOMMITDATEOVERRIDE } Else { $LastCommitOlderThan } 
+    if (!$res.object -or !$res.object.url) {
+      LogDebug "No commit url returned from response. Skipping... "
+      continue
+    }
+    try {
+      $commitResponse = Invoke-WebRequest $res.object.url
```

Comment:
We should add a helper in the Invoke-GithubAPIs something like get commit date and we should have that return committer.date instead of author.date.

Reply:
Also I think we want to call Invoke-RestMethod instead of Invoke-WebRequest